src/Spd.Manager.Licence/SharedContract.cs

```diff
@@ -89,6 +89,7 @@ public enum PoliceOfficerRoleCode
     CourtAppointedBailiff,
     SpecialProvincialOrMunicipalConstable,
     PoliceOfficer,
+    PoliceOfficerRetired,
     Other,

```

Comment:
do we need to add code in read, if it returned None, then we return null?

Reply:
the same for create, if user input null, should we set it to None in dynamics?

Reply:
I removed None form both `PoliceOfficerRoleCode` and `PoliceOfficerRoleEnum` enums.
just have None in optionset, which in the setting and getting function I've handled it

*fe sends and receives null, however the source is `null` or `None`
@carolcarpenter

Reply:
please also deal with the oppsite direction, if fe set to null, then be needs to set None in dynamics.

Reply:
it is doing this way now, just save None, never saves or updates to null in db